problem-spring-web/src/test/java/org/zalando/problem/spring/web/advice/security/SecurityAdviceTraitTest.java

```diff
@@ -45,12 +47,16 @@
 @WebAppConfiguration
 final class SecurityAdviceTraitTest {
 
+    @BeforeEach
+    void setUp() {
+        Locale.setDefault(Locale.ENGLISH);
```

Comment:
I played around and couldn't get anything to work using `LocaleResolver`. What mitigates my concerns a bit would be doing this:

```java
@BeforeEach
void setUp() {
    LocaleContextHolder.setLocale(Locale.ENGLISH);
}

@AfterEach
void tearDown() {
    LocaleContextHolder.resetLocaleContext();
}
```

Reply:
That one does the trick in the surefire plugin configuration.

```xml
<argLine>-Duser.language=en</argLine>
```

Works well in IntelliJ - it's immediately synchronized when modified and applied to tests being run in the IDE itself.

Reply:
Imho all the test fixture (for me in this case it includes the locale) should be in the test itself and not in the build scripts.
I updated to the LocaleContestHolder approach.